android/src/main/java/com/swmansion/reanimated/layoutReanimation/ReanimatedNativeHierarchyManager.java

```diff
@@ -206,6 +224,13 @@ public synchronized void manageChildren(
       @Nullable int[] tagsToDelete) {
     ViewGroup viewGroup = (ViewGroup) resolveView(tag);
     ViewGroupManager viewGroupManager = (ViewGroupManager) resolveViewManager(tag);
+
+    // we don't want layout animations in native-stack since it is currently buggy there
+    if (viewGroupManager.getName().equals("RNSScreenStack")) {
```

Comment:
It may be safe to check if `viewGroupManager` is not `null`? Or is there no such option?